bedrock/base/templates/macros.html

```diff
@@ -522,7 +522,6 @@
           <ul class="subnav" id="os-subnav">
             <li><a data-id="os-index" href="{{ url('firefox.os.index') }}"{% if activesub == 'index' %} class="selected"{% endif %}><div>{{ _('Overview') }}</div></a></li>
             <li><a data-id="os-devices" href="{{ url('firefox.os.devices') }}"{% if activesub == 'devices' %} class="selected"{% endif %}><div>{{ _('Devices') }}</div></a></li>
-            <li><a data-id="os-partners" href="{{ url('firefox.partners.index') }}"{% if activesub == 'partners' %} class="selected"{% endif %}><div>{{ _('For partners') }}</div></a></li>
```

Comment:
Super duper small nit - update instructions on the macro to not include `partners` as a valid sub nav ID for `os`.